Local installation of swagger-preview

nisalVD opened the pull request:
Hi,

heres a few things i did in this PR

- added a package.json to the package, and mapped out `swagger-ui-watcher` locally instead of a globally installed node module. 

- added documentation for lazy.nvim

- fixed a typo in documentation

vinnymeller commented on README.md:
````diff
@@ -10,14 +10,25 @@ A neovim plugin to live-preview Swagger files.
+```lua
+
+    return {
+      "nisalVD/swagger-preview.nvim",
````
```suggestion
      "vinnymeller/swagger-preview.nvim",
```

vinnymeller commented on README.md:
```diff
@@ -10,14 +10,25 @@ A neovim plugin to live-preview Swagger files.
+
+    return {
+      "nisalVD/swagger-preview.nvim",
+      cmd = { "SwaggerPreview", "SwaggerStop", "SwaggerPreviewStop" },
```
should these line up with the commands provided by the plugin?

```suggestion
      cmd = { "SwaggerPreview", "SwaggerPreviewStop", "SwaggerPreviewToggle" },
```

vinnymeller reviewed (commented):
couple small things otherwise looks good / seems to be working

nisalVD commented:
Hey thanks for that, added your changes

added a `.stylua.toml` to keep the formatting consistent, happy to change that to whatever you prefer for now i just used whats used in telescope.nvim
